test/demo/utils/ofPuppeteer.ts

```diff
@@ -0,0 +1,141 @@
+import {Fin, Identity} from 'openfin/_v2/main';
+import {Browser, Page, JSHandle} from 'puppeteer';
+import {connect} from 'hadouken-js-adapter';
+
+declare const global: NodeJS.Global & {__BROWSER__: Browser};
+
+export type BaseWindowContext = Window & {fin: Fin};
+
+export class OFPuppeteerBrowser<WindowContext extends BaseWindowContext = BaseWindowContext> {
```

Comment:
A bit of clean-up, can we order members to help with readability/consistency?

```
static vars (public -> protected -> private)
static functions (public -> protected -> private)

member vars (public -> protected -> private)

constructor

getters/setters (public -> protected -> private)
member functions (public -> protected -> private)
```

Reply:
done